src/ResourceManagement/ServiceFabric/Microsoft.Azure.Management.ServiceFabric/generate.cmd

```diff
@@ -0,0 +1,17 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+set autoRestVersion=1.0.0-Nightly20170202
+if  "%1" == "" (
+    set specFile="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-servicefabric/2016-09-01/swagger/servicefabric.json"
```

Comment:
@chingchenmsft please change the URL to point at the commit you used, rather than the branch. I believe you would end up using [this URL](https://github.com/Azure/azure-rest-api-specs/blob/13dee0c45f4ac20ba3f339f367e944bcbba06ffd/arm-servicefabric/2016-09-01/swagger/servicefabric.json).